tests/models/test_domain_registries.py

```
@@ -0,0 +1,309 @@
+        old_model.generate = MagicMock(return_value="response")
+        mgr._active_models["text"] = old_model
+
+        errors: list[Exception] = []
+
+        def generate_loop() -> None:
+            for _ in range(10):
+                try:
+                    model = mgr.get_active_model("text")
+                    if model and hasattr(model, "generate"):
+                        model.generate("test prompt")
+                except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)